### ISSUE TITLE
The webview find widget is focusable while hidden

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#67563]
Microsoft/vscode#67509

Repro:

1. Open PR ext description page
2. Focus something, shift+tab until you hit the start of the screen

There are 4 focusable items, close, next match, previous match and the find input.

**Comment by cleidigh** [comment]
@mjbvz 
Would you like  me to start on this?

**Comment by mjbvz** [comment]
Yes please. It is too late to get a fix in for the January VS Code release but you can submit a PR any time and we'll try to get this fixed for February

**Comment by cleidigh** [comment]
LOL ... my Dragon dictation did the equivalent of a butt dial , deleted assignments and pasted an eMoji

sorry about that :-P

**Comment by cleidigh** [comment]
@Tyriar 
@mjbvz 
FYI:  started on this and I can reproduce and see it.  

- simpleFind transitions in and out with animation, hidden only because of z-index and/or overflow.
- The widget always has display type:flex , therefore the four tabs stops for the buttons and input are always in the tabIndex order
- fix should be simple, however, there is an interaction between the animation and the display mode
  such that simultaneously going from display:none => flex messes up the animation.  I'm working on separating the display mode from the transition to see if that will work.

I have to do more experiments , May have a question or two later...

**Comment by cleidigh** [comment]
See PR: #68463

Verification:
Use a screen reader with screen output on so you can see and track each tab stop

- Open the new pull request extension page
- Click on the upper pane to begin focus there
- Tab through the various toolbar buttons , after dependencies the next tab SHOULD land on
  With the webview not the simpleFind widget.

NVDA Output:  with visibility fix (correct output, no tab stops within the hidden simpleFind)

Extension: GitHub Pull Requests - vscode - Code - OSS Dev  document

`` 
tool bar
Disable▼  button
Uninstall  button
tool bar
Contributions  button  Lists contributions to VS Code by this extension
Changelog  button  Extension update history, rendered from the extension's 'CHANGELOG.md' file
Dependencies  button  Lists extensions this extension depends on
frame
frame
scroll to top
button  data:text/html;charset=utf-8,%3C%21DOCTYPE%20html%3E%0D%0A%3Ch.... (truncated)
graphic
Extension: GitHub Pull Requests - vscode - Code - OSS Dev  document
button
scroll to top
``

NVDA Output: original scenario without fix (can see tabs stops in widget)

``
the marketplace
link  https://aka.ms/vscodepr-download
Extension: GitHub Pull Requests - vscode - Code - OSS Dev
tool bar
Disable▼  button
Uninstall  button
tool bar
Contributions  button  Lists contributions to VS Code by this extension
Changelog  button  Extension update history, rendered from the extension's 'CHANGELOG.md' file
Dependencies  button  Lists extensions this extension depends on
Find  edit  Find
Find
blank
Previous match
button  Previous match
Next match
button  Next match
Close
button  Close
frame
frame
list  with 7 items
the marketplace
link  https://aka.ms/vscodepr-download
scroll to top
button  data:text/html;charset=utf-8,%3C%21DOCTYPE%20html%3E%0D%0A%3Chtml%20lang%3D%22en%22%20style%3D%22width%3A%20100%25%3B%20height%3A%20100%25%22%3E%0D%0A%3Chead%3E%0D%0A%09%3Ctitle%3EVirtual%20Document%3C%2Ftitle%3E%0D%0A%3C%2Fhead%3E%0D%0A%3Cbody%20style%3D%22margin%3A%200%3B%20overflow%3A%20hidden%3B%20width%3A%20100%25%3B%20height%3A%20100%25%22%3E%0D%0A%3C%2Fbody%3E%0D%0A%3C%2Fhtml%3E#
list  with 7 items

``
